README.md

```diff
@@ -165,6 +163,16 @@ cmake -DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ -D
 make check
 ```
 
+## Docker images
+| Docker files | Image | Tags | Description |
+|---------------|------|-----|--|
+| [./Dockerfile-prover](./Dockerfile-prover) | [clearmatics/zeth-prover](https://hub.docker.com/r/clearmatics/zeth-prover) | `latest`, `vX.Y.Z` - Release of zeth, `git-%HASH%` - developers build by git-commit  | [Zeth Prover Server](./prover_server/README.md). Image use `zeth-base` for building |
+| [./Dockerfile-client](./Dockerfile-client) | [clearmatics/zeth-client](https://hub.docker.com/r/clearmatics/zeth-client) | `latest`, `vX.Y.Z` - Release of zeth, `git-%HASH%` - developers build by git-commit  | [Python client to interact with the prover](./client/README.md) |
+| [./Dockerfile-mpc](./Dockerfile-mpc) | [clearmatics/zeth-mpc](https://hub.docker.com/r/clearmatics/zeth-mpc) | `latest`, `vX.Y.Z` - Release of zeth, `git-%HASH%` - developers build by git-commit  | [Tools for Multi-Party Computation](./mpc/README.md). Image use `zeth-base` for building |
+| [./Dockerfile-base](./Dockerfile-base) | [clearmatics/zeth-base](https://hub.docker.com/r/clearmatics/zeth-base) | `latest`, `vA.B.C` - Release of zeth-base | Base image for building other containers |
```

Comment:
`vA.B.C` -> `vX.Y.Z`? For consistency or did you use `A.B.C` and `X.Y.Z` explicitly to represent the fact that the versioning of `zeth-base` and of the rest of the project can be independent?

Reply:
Yes, zeth-base and zeth-apps is different products just placed in one repo. Release cycle is different.

Available options:
* Build all artefacts as one product sequently for each commit (it takes ages) or
* Split the products releases

Reply:
Thanks, makes sense